packages/video-intelligence/src/index.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ 'use strict';
+
+ var gapic = {
+   v1beta1: require('./v1beta1'),
```

Comment:
It's a best practice to reference the direct file, saving some overhead at runtime-- not sure of the specifics, but I read of this being necessary going forward when more robust module loaders are available. In this case, I think we want `v1beta1/index.js`.